assets/images/coffeescript.svg

```diff
@@ -0,0 +1,6 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Are we actually cool with putting the Mozilla license header on images of these library logos? I swear some may have trademarking or something.

Reply:
I'm no lawyer but afaik it's OK to use trademarks as long as you're not misleading people into thinking that you're associated with the company who owns it.